package.json

```diff
@@ -10,8 +10,11 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "hash-router": "^0.4.0",
```

Comment:
@krishna1212031 Please uninstall this package. This is not needed.
```npm uninstall hash-router```

Instead of this, add HashRouter in app.tsx file.
Refer this: [Hash Router](https://reactrouter.com/web/api/HashRouter)